federation/cluster/federation-up.sh

```diff
@@ -90,7 +90,7 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
+      --apiserver-arg-overrides="--storage-backend=etcd2,--runtime-config=api/all=true" \
```

Comment:
Yes. `apiserver-arg-overrides` already exists in kubefed. We should use that in the tests to set runtime-config. We do not want alpha apis to be enabled by default. Need to enable them explicitly in tests.